packages/syntax-core/src/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,126 @@
+import classNames from "classnames";
+import backgroundColor from "../colors/backgroundColor";
+import foregroundColor, {
+  foregroundTypographyColor,
+} from "../colors/foregroundColor";
+import React from "react";
+import { type Color, type Size } from "../constants";
+import Typography from "../Typography/Typography";
+
+import buttonStyles from "../Button/Button.module.css";
+import { textVariant, iconSize } from "../Button/ButtonConstants";
+
+import styles from "./LinkButton.module.css";
+
+/**
+ * LinkButton is a "variation" of Button that should look identical to Button, but should be used to render links instead.
```

Comment:
@somethiiing Could you also add a link to the docs like we do now with our other components?

https://github.com/Cambly/syntax/blob/4fb87f34874c47cac6e037f5d77a4ba443e05a76/packages/syntax-core/src/TextField/TextField.tsx#L12